Gemfile

```diff
@@ -1,7 +1,6 @@
 source 'https://rubygems.org' do
   gem 'rake'
-  gem 'cocoapods', '~> 1.8'
-  gem 'cocoapods-repo-update', '~> 0.0.3'
+  gem 'cocoapods', '>= 1.10'
```

Comment:
CocoaPods 1.10.1 is required to properly build the `xcframework`s that Sodium (a Tracks dependency) uses to support Apple Silicon.

The repo update plugin is no longer necessary because CocoaPods uses the CDN by default.